docs/sdkx-react-native/release-notes-react-native.mdx

```diff
@@ -21,6 +21,13 @@ import {
 
 # Release Notes
 
+## 10.2.3
+
+_May 3rd, 2023_
+
+** Release Notes: **
+
+- Added `HelpshiftLog`, public api for additional debug logs, please refer [here](/sdkx-react-native/tracking/#debug-logs).
```

Comment:
Added an API for attaching debug logs to a new conversation filed by an end-user. For more details, refer our [Tracking](/sdkx-react-native/tracking/#debug-logs) page.